src/software/gui/geometry_conversion/geometry_conversion.cpp

```diff
@@ -1,4 +1,4 @@
-#include "software/gui/visualizer/geom/geometry_conversion.h"
+#include "software/gui/geometry_conversion/geometry_conversion.h"
```

Comment:
Why is this in it's own folder now? if it's just one library I think it's fine for it to just live in `src/software/gui`?

Reply:
It was in case it sort of became like a `util` folder, which I thought is might since I didn't know all of what I needed to add. Happy to change if you think it's worth it.

Reply:
Yeah, I think we should pull it up if we only have one lib for now.

Reply:
done